draft-ietf-mls-architecture.md

```diff
@@ -1908,11 +1911,7 @@ In certain cases, the adversary can access specific bindings between public keys
 and identities. If the signature keys are reused across groups, the adversary
 can get more information about the targeted user.
 
-> **RECOMMENDATION:** Do not use the same signature keypair across
-> groups. Update all keys for all groups on a regular basis. Do not preserve
-> keys in different groups when suspecting a compromise.
-
-> **RECOMMENDATION:** Separate the service binding the identities and the public
+> **RECOMMENDATION:** Separate the binding the identities and the public
```

Comment:
s/Separate the binding the identities and the public/Separate the binding of the identities and the corresponding public/